src/main/java/com/linkedin/kmf/apps/MirrorPipelineMonitor.java

```diff
@@ -70,6 +70,7 @@ public MirrorPipelineMonitor(Map<String, Object> props, String name) throws Exce
                                                                      String topic,
                                                                      String name) throws Exception {
     Map<String, Object> topicManagementProps = createTopicManagementServiceProps(props, topic);
+    topicManagementProps.put(CommonServiceConfig.ZOOKEEPER_CONNECT_CONFIG, "");
```

Comment:
A zookeeper_connect_config is a requirement for TopicManagementServiceConfig

https://github.com/linkedin/kafka-monitor/blob/master/src/main/java/com/linkedin/kmf/services/configs/TopicManagementServiceConfig.java